debezium-api/src/main/java/io/debezium/engine/format/Change.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.engine.format;
+
+public class Change<T> {
```

Comment:
I'll push a commit to rename it to `ChangeEvent`; a bit more meaningful and in line with the term we use in blogs, presentations etc.

Slightly more problematic: it assumes that key and value are of the same type, I don't think that generally holds. I think we need to have two type parameters here, for key and value. Note the bootstrap method for getting an engine could be overloaded, i.e. there would be the one with one parameter, so to use the same converter for both key and value, but there'd be another one, allowing for two separate converters.